src/redux/configureStore.js

```diff
@@ -2,9 +2,11 @@ import { combineReducers, createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import logger from 'redux-logger';
 import rockets from './rockets/rockets';
```

Comment:
The name of the rockets reducer was changed by me for clarity.
It was causing merge conflicts, but it was already solved.